README.md

```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.org/containernetworking/cni.svg?branch=master)](https://travis-ci.org/containernetworking/cni)
+[![Linux Build Status](https://travis-ci.org/containernetworking/cni.svg?branch=master)](https://travis-ci.org/containernetworking/cni)
+[![Windows Build Status](https://ci.appveyor.com/api/projects/status/e6jkofdn81d3ey1q/branch/master?svg=true)](https://ci.appveyor.com/project/rosenhouse/cni-caa6b)
```

Comment:
is this linked to a personal ('rosenhouse') account?

Reply:
yeah....it's a personal appveyor account, but linked to the real upstream github source repo.

appveyor doesn't appear to support github orgs being owners of appveyor projects.  one option is we can create a "cni-ci-bot" account on github, and sign that up for an appveyor account.  then us maintainers need to share ownership of the credentials for that github account.

thoughts?